packages/web-runtime/src/components/Topbar/TopBar.vue

```diff
@@ -125,14 +138,10 @@ export default {
       return props.applicationsList
         .filter((app) => {
           if (app.type === 'extension') {
-            // check if the extension has at least one navItem with a matching menuId
             return (
-              store.getters
-                .getNavItemsByExtension(app.id)
-                .filter((navItem) => isNavItemPermitted(permittedMenus, navItem)).length > 0 ||
-              (app.applicationMenu.enabled instanceof Function &&
-                app.applicationMenu.enabled(store, ability) &&
-                !permittedMenus.includes('user'))
+              app.applicationMenu.enabled instanceof Function &&
```

Comment:
Hmm do we still need the check if it's a function? It's correctly typed as a function, so I guess we can assume that here.

Reply:
unfortunately yes, applicationMenu will  be always present, see `packages/web-runtime/src/store/apps.ts`
so unfortantly optional chaining also doesn't work, as you will call object.undefinted(x,y)